redact/v4/tools/redact_folder.py

```diff
@@ -37,14 +38,16 @@ def redact_folder(
     service: ServiceType,
     job_args: Optional[JobArguments] = None,
     licence_plate_custom_stamp_path: Optional[str] = None,
-    redact_url: str = settings.redact_url_default,
+    redact_url: Union[str, List[str]] = [settings.redact_url_default],
```

Comment:
Hi @pkern90 , here you can see that the new multi-url feature for client-side load balancing was implemented in a  backwards-compatible fashion.